### PR TITLE
Read the fine print. - Ready for merge

### PR DESCRIPTION
**Pull request by ogmage78**
UPDATE 10/21
**Fixed all outstanding issues**

* Added processing for contract management add / abandon.
* Removed update script I added back on 10/14 to fix the burden of the weenie for the MMD note.   This is no longer needed - use Dereth v0.2.4 or greater the issue has been  resolved in the data and the script is no longer needed.
* Minor code cleanup and improved some method documentation.
* Added new table to store persisted tracked contracts (quests)
* Implemented two new game events / SendClientContractTrackerTable and SendClientContractTracker
* Implemented abandon quest - Game action - Social_AbandonContract
* Documented methods and updated trello.
*To test @ci 44163 -create a contract.   Use the contract and you are now tracking that quest.   You can create any contract using ci or buy one from a vendor if they are In I did not check vendors.   You can abandon the contract.   All management and the cool down delay of 2 seconds is respected.

download

### CODE REVIEW

File: Source/ACE/Entity/Player.cs [hunk 1]
```diff
@@ -63,20 +62,7 @@ public enum ConsumableBuffType : uint
         /// </summary>
         private Dictionary<ObjectGuid, WorldObject> interactiveWorldObjects = new Dictionary<ObjectGuid, WorldObject>();
 
-        /// <summary>
```
Comment (Mogwai-TheFurry) [inline Source/ACE/Entity/Player.cs]: what happened to these 3 properties?  were they not needed?

File: Source/AppVeyor/MySqlInstall.bat [hunk 2]
```diff
@@ -19,5 +19,5 @@ appveyor DownloadFile https://github.com/ACEmulator/ACE-World/releases/download/
 
 REM execute Update Scripts for World Database
 "C:\Program Files\MySql\MySQL Server 5.7\bin\mysql.exe" -h localhost -u root -pPassword12! ace_world < database\updates\world\06-06-30-2017-generator-chains-testdata.sql
-"C:\Program Files\MySql\MySQL Server 5.7\bin\mysql.exe" -h localhost -u root -pPassword12! ace_world < database\updates\world\10-14-2017-fix-weenie-mmd-20630.sql
+"C:\Program Files\MySql\MySQL Server 5.7\bin\mysql.exe" -h localhost -u root -pPassword12! ace_shard < database\updates\shard\10-18-2017-create-ace-object-tracker.sql
```
Comment (Mogwai-TheFurry) [inline Source/AppVeyor/MySqlInstall.bat]: is the fix-weenie script no longer needed?

File: Source/ACE/Entity/Player.cs (hunk 1 above)
Comment (ogmage78) [reply Source/ACE/Entity/Player.cs]: They were stuff I had added 6 months ago and had no references.    I think they were added before I knew what was going on.

File: Source/AppVeyor/MySqlInstall.bat (hunk 2 above)
Comment (ogmage78) [reply Source/AppVeyor/MySqlInstall.bat]: No Ripley latest fixes the issue.   I put a notice of the version in change log.

**Pull request merged**